appintro/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_intro_skip_button">PULAR</string>
```

Comment:
Hey @GuilhE 
I assume you skipped the other strings as they're the same in portuguese and `pt-rBR` right?

Reply:
Yup exactly :)